pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -127,10 +173,11 @@ type ReconcileHandler interface {
 type ReconcileNodeMaintenance struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
-	client      client.Client
-	scheme      *runtime.Scheme
-	drainer     *drain.Helper
-	podInformer cache.SharedInformer
+	client    client.Client
+	scheme    *runtime.Scheme
+	drainer   *drain.Helper
+	podInformer      cache.SharedInformer
```

Comment:
is podInformer still in use here?

Reply:
shouldn't be
something weird is happening with the alignment too

Reply:
podinformer is still used in this PR, it has been removed in PR #60, but this PR doesn't touch the podInformer.

Reply:
removed the whitespace diff.